requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest==7.1.1
+pytest-testinfra==6.0.0
+PyYAML==6.0
```

Comment:
We should rework this with [the requirements.in and `pip-compile` process we use in astronomer/astronomer](https://github.com/astronomer/astronomer/blob/f8eb4bbab437a7/Makefile#L60-L63) so the upgrade process is easier and more reliable.

Reply:
Updated to use `pip-tools` as suggested.